README.md

```diff
@@ -14,6 +14,27 @@ It was written to follow along with the book page by page and catalogue dependen
 
 The PDF file of the book available from Paul Graham's site isn't so great for reading on a screen. A version with smaller margins and the missing figures re-added can be found [here](http://www.lurklurk.org/onlisp/onlisp.html).
 
-An online HTML version of the book, minus graphical figures, can be found [here](http://dunsmor.com/lisp/onlisp/onlisp.html).
+An online HTML version of the book, minus graphical figures, can be found [here](http://lisp.esthlos.com/onlisp/on_lisp.html).
```

Comment:
The book index links are broken in this version, but the HTML book itself appears to be intact